GOVERNANCE.md

```diff
@@ -0,0 +1,87 @@
+- [Becoming a Maintainer](#becoming-a-maintainer)
+- [Meetings](#meetings)
+- [Code of Conduct Enforcement](#code-of-conduct)
+- [Modifications](#modifying-this-charter)
+
+## Values
+
+The ModelPack project leadership embraces the following values:
+
+- **Openness**: Communication and decision-making happens in the open and is discoverable for future reference. As much as possible, all discussions and work take place in public forums and open repositories.
+
+- **Fairness**: All stakeholders have the opportunity to provide feedback and submit contributions, which will be considered on their merits.
+
+- **Community over Product or Company**: Sustaining and growing our community takes priority over shipping code or sponsors' organizational goals. Each contributor participates in the project as an individual.
+
+- **Inclusivity**: We innovate through different perspectives and skill sets, which can only be accomplished in a welcoming and respectful environment.
+
+- **Participation**: Responsibilities within the project are earned through participation, and there is a clear path up the contributor ladder into leadership positions.
+
+## Maintainers
+
+ModelPack maintainers have write access to the [project GitHub repository](https://github.com/modelpack/model-spec). They can merge patches and are expected to review large or critical patches personally. The current maintainers can be found in [MAINTAINERS.md](./MAINTAINERS.md). Maintainers collectively manage the project's resources and contributors.
+
+This privilege is granted with some expectation of responsibility: maintainers are people who care about the ModelPack project and want to help it grow and improve. A maintainer is not just someone who can make changes, but someone who has demonstrated good judgement and an ability to collaborate with the team, get the most knowledgeable people to review code and docs, contribute high-quality code, and follow through to fix issues (in code or tests).
+
+A maintainer is a contributor to the project's success and a citizen helping the project succeed.
+
+The collective team of all Maintainers is known as the **Maintainer Council**, which is the governing body for the project.
+
+### Becoming a Maintainer
+
+To become a Maintainer you need to demonstrate the following:
+
+- **Commitment to the project**:
+  - Participate actively and impactfully in discussions, contributions, code and documentation reviews `this will need quantifying eventually`
+  - Perform useful reviews for a significant number of non-trivial pull requests `this will need quantifying eventually`
+  - Contribute non-trivial pull requests and have them merged `this will need quantifying eventually`
+- Ability to write quality code and/or documentation
+- Ability to collaborate with the team
+- Understanding of how the team works (policies, processes for testing and code review, etc)
+- Understanding of the project's code base and coding and documentation style
```

Comment:
Should we also include any mention of interfacing with the CNCF?

Reply:
These are general ability requirements for a maintainer. It is more of a task to interface with the CNCF. So let's not be specific here.